addons/test_website/tests/test_fuzzy.py

```diff
@@ -96,3 +96,18 @@ def test_02_pages_search(self):
                     'displayImage': False, 'allowFuzzy': True
                 }
             )
+
+    def test_indirect(self):
+        self._autocomplete('module', 1, 'model')
+        self._autocomplete('parental', 1, 'partial', {
+            'force_detail': {
+                'search_fields': ['name', 'sub_ids.name'],
+            }
```

Comment:
Is there a reason to not add those `'sub_ids.name'` and `''sub_ids.tag_id.name'` fields directly in the `_search_get_detail` method? Without tweaking the test methods with `force_detail`?
Wouldn't we do it that way if we wanted a model to also consider a subfield? It would be the responsibility of the callee (`_search_get_detail`), not the caller, right?
I am maybe wrong, you know this part of the code better than I do